_posts/2025-07-25-A-real-use-case-with-OVOS-and-Hivemind.md

```diff
@@ -0,0 +1,59 @@
+
+Looking ahead, the next step is integrating a personal AI assistant directly into the voice assistant. This enables a private, fully offline solution—something that’s becoming increasingly important for privacy, reliability, and autonomy.
+
+### How did I built a Smarter Living with OVOS, Hivemind, and Raspberry Pi Satellites
+
+In this showcase, I’ve built a basic voice setup using Open Voice OS (OVOS) and Hivemind. The OVOS server runs on a compact Intel NUC with a 5th-gen i5 processor. It hosts both the OVOS core and a Generative AI model (Ollama running Gemma3:1B), which enables offline voice control and (some) intelligence and context-aware responses. By linking OVOS to my Homey gateway, I can control lighting, audio, and various smart devices with natural speech.
+
+To expand voice coverage throughout the house, I added Raspberry Pi Zero 2W units as satellites. These low-power devices use local voice activation (VAD) to detect the wakeword ("Hey Mycroft"). Once triggered, they record the spoken command and forward the audio to the Hivemind listener on the OVOS server. Hivemind then routes the audio to OVOS, which parses the utterance and executes the appropriate intent—whether it’s starting a radio stream, sending a message to the AI assistant, or toggling a device via the Homey API.
+
+While OVOS and Hivemind are still evolving, this setup already proves itself in real-world use. It’s stable enough for friendly testing and flexible enough to grow with future needs.
+
+## The Show case
+
+The showcase is using Dutch language. Trust me, the system does what it should do.
+Satellites: Components https://github.com/MenneBos/ovos-skill-HomeyFlowTrigger/blob/main/Media/Sat_components.jpg
+Satellites: Assembled https://github.com/MenneBos/ovos-skill-HomeyFlowTrigger/blob/main/Media/Sat_assembled.jpg
+Satellite: In kitchen https://github.com/MenneBos/ovos-skill-HomeyFlowTrigger/blob/main/Media/Sat_kitchen.jpg
+Satellite: In sleeping room https://github.com/MenneBos/ovos-skill-HomeyFlowTrigger/blob/main/Media/Sat_sleepingroom.jpg
+Controlling Curtains in sleeping room [video]: https://github.com/MenneBos/ovos-skill-HomeyFlowTrigger/blob/main/Media/Open_Curtain_SleepingRoom.mp4
+Controlling the coffee machine [video]: https://github.com/MenneBos/ovos-skill-HomeyFlowTrigger/blob/main/Media/Coffee_automation.mp4
+Useing the AI assistant [video]: https://github.com/MenneBos/ovos-skill-HomeyFlowTrigger/blob/main/Media/GenAI_assistant.mp4
+
+### Including Images
+
+More information on OVOS and Hivemind:
+OVOS: https://ovoshatchery.github.io/community-docs/
+Hivemind: https://jarbashivemind.github.io/HiveMind-community-docs/
+
+## Conclusion
+
+OVOS has key feature for controlling the house:
+- fine control on intents to enable predictive outcomes
+- satellite architecture for flexibility
+- [future] Integrated with local or private GenAI
+
+👉 [Support the project here](https://www.openvoiceos.org/contribution)
```

Comment:
maybe copy the call to action snippet we have been used in other blog posts for consistency, its a bit more fleshed out than a single link